README.md

```diff
@@ -1,3 +1,14 @@
 Testing example for begginers
 
+S-Single responsibility principle: This concept say that every class only need must 	will have one job, that’s mean that the system should be separate in 		different modules and everyone should will have a job. That help to make 		easier at the moment when in necessary change the code, because the module 		that need the change is the only that will change.
```

Comment:
This concept say that every class only will have one job. The 'need' and 'must' words are not necessary here. Please fix this and commit it again.